### PR TITLE
Refactoring to work on #258

### PR DESCRIPTION
**Pull request by strengejacke**

### CODE REVIEW

File: R/report_htest_ttest.R [hunk 1]
```diff
@@ -78,3 +78,38 @@
...
 }
+
+
+# report model ---------------------------
+
+.report_model_ttest <- function(x, table) {
+  # If against mu
+  if (names(x$null.value) == "mean") {
+
+    # TODO: @DominiqueMakowski why do we need "table" here?
```
Comment (strengejacke) [inline R/report_htest_ttest.R:89]: ping @DominiqueMakowski

**Comment by strengejacke** [comment]
ok, I think the refactoring is completed now, including some performance improvements (reduce multiple calls to `model_info()`). We can now go on and address #258 and similar.

**Comment by IndrajeetPatil** [comment]
Thanks, Daniel, for the refactoring!

I think we should merge this before we can start working on #258 to keep the scope of the PR narrow.

**Review by IndrajeetPatil — APPROVED** [review]

File: R/report_htest_ttest.R (hunk 1 above)
Comment (DominiqueMakowski) [reply R/report_htest_ttest.R:89]: I think it was for standardization ? can't remember though

**Review by DominiqueMakowski — APPROVED** [review]

**Comment by DominiqueMakowski** [comment]
Looks good to me, the breakdown of into smaller chunks should make it easy to adapt it for {correlation}'s `cor_test()`

**Pull request merged**